src/Component/Filter/AttributeCombo/AttributeCombo.tsx

```diff
@@ -71,7 +70,7 @@ class AttributeCombo extends React.Component<AttributeComboProps, AttributeCombo
     return (
       <div className="gs-attr-combo">
 
-        <FormItem label={this.props.label} colon={false} >
+        <Form.Item label={this.props.label} colon={false} >
```

Comment:
Is this change required for this PR to work. If not please split this into another PR.